ui/components/config/mcp-client-form.tsx

```
@@ -1,342 +1,342 @@
-						}
-					: undefined,
-			tools_to_skip: parseArrayFromText(toolsToSkipText),
-			tools_to_execute: parseArrayFromText(toolsToExecuteText),
-		};
+    // Prepare the payload
+    const payload: CreateMCPClientRequest = {
+      ...form,
+      stdio_config:
+        form.connection_type === 'stdio'
+          ? {
+              ...form.stdio_config!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)